src/Umbraco.Web/Mvc/RenderIndexActionSelectorAttribute.cs

```diff
@@ -7,7 +7,7 @@
 namespace Umbraco.Web.Mvc
 {
     /// <summary>
-    /// A custom ActionMethodSelector which will ensure that the RenderMvcController.Index(RenderModel model) action will be executed
+    /// A custom ActionMethodSelector which will ensure that the RenderMvcController.Index(ContentModel model) action will be executed
     /// if the
```

Comment:
@kjac I think it would be great to re-write this sentence since it doesn't finish the sentence it just stops with `... if the` ... if  what?? 😂

Reply:
Haha! I didn't even see that... I guess @Shazwazza needs to explain it. If he can remember it... the original comment is from November 2015 😆